me5413_perception:
  # the frame of all the poses below
  frame_id: 'map'

  # robot starting point
  door_entrance:
    position:
      x: 16.0
      y: 1.0
      z: 0.0
    orientation:
      x: 0.0
      y: 0.0
      z: -0.9
      w: 0.23

  door_exit:
    position:
      x: 7.5
      y: -6.5
      z: 0.0
    orientation:
      x: 0.0
      y: 0.0
      z: 0.45
      w: 0.9